apps/meteor/app/cors/server/cors.ts

```diff
@@ -77,6 +79,17 @@ WebApp.rawConnectHandlers.use((_req: http.IncomingMessage, res: http.ServerRespo
 });
 
 const _staticFilesMiddleware = WebAppInternals.staticFilesMiddleware;
+declare module 'meteor/webapp' {
+	// eslint-disable-next-line @typescript-eslint/no-namespace
+	namespace WebApp {
+		function categorizeRequest(
+			req: http.IncomingMessage,
+		): { arch: string; path: string; url: UrlWithParsedQuery } & Record<string, unknown>;
+	}
+}
+
+// These routes already handle cache control on their own
+const cacheControlledRoutes: Array<string> = ['/assets', '/custom-sounds', '/emoji-custom', '/avatar', '/file-upload'];
```

Comment:
if we have the regexps already on this array, we don't need to do a `new RegExp` on every request:

```suggestion
const cacheControlledRoutes: Array<string> = ['/assets', '/custom-sounds', '/emoji-custom', '/avatar', '/file-upload'].map((route) => new RegExp(`^${route}`, 'i'));
```

Reply:
I was using the global option before, and I know that running the same regex with the global option enabled can have problems. I just realized I don't actually need the 'g' flag 🙈 . Thanks for the tip.